src/commands/lambda/instrument.ts

```diff
@@ -177,15 +182,46 @@ export class InstrumentCommand extends Command {
     const tracingEnabled = this.convertStringBooleanToBoolean(true, this.tracing, this.config.tracing)
     const logLevel = this.logLevel ?? this.config.logLevel
 
+    const service = this.service ?? this.config.service
+    const environment = this.environment ?? this.config.environment
+    const version = this.version ?? this.config.version
+
+    const tagsMap: {[key: string]: string | undefined} = {
+      environment,
+      service,
+      version,
+    }
+    let tagMissing = false
+    for (const [tag, value] of Object.entries(tagsMap)) {
+      if (!value) {
+        tagMissing = true
+        this.context.stdout.write(`No value found for the ${tag} tag.\n`)
+      }
+    }
+    if (tagMissing) {
+      this.context.stdout.write('It is recommended to set tags, try to do it later.\n')
```

Comment:
I think we need to warn the customer only if service/env/version is missing, tags should remain optional

Reply:
Oh yeah, that code is from a map that contains those, maybe should just rename the map and the boolean